amf-client/shared/src/test/resources/validations/reports/model/invalid-oas-required.report

```diff
@@ -1,21 +1,31 @@
 Model: file://amf-client/shared/src/test/resources/validations/invalid-oas-required/invalid-oas-required.json
 Profile: OAS 2.0
 Conforms? false
-Number of results: 2
+Number of results: 3
 
 Level: Violation
 
+- Source: http://a.ml/vocabularies/amf/parser#description-is-required-in-response
```

Comment:
This was added because this api is now not cut before validation. As it had parsing violations before it never got to the validation phase.